src/Http.ts

```diff
@@ -14,6 +15,9 @@ export default class Http {
       baseURL: this.host + 'api/v2/storefront',
       headers: {
         'Content-Type': 'application/vnd.api+json'
+      },
+      paramsSerializer: (params) => {
+        return qs.stringify(params, {arrayFormat: 'brackets'})
```

Comment:
add space between brackets it should be ` return qs.stringify(params, { arrayFormat: 'brackets' })`

Reply:
spaces added